torch/fx/symbolic_trace.py

```diff
@@ -125,8 +125,12 @@ def create_args_for_root(self, root_fn, is_module):
             next(names_iter)  # skip self
             args.append(self.root)
 
+        sig = inspect.signature(fn_for_analysis)
+
         def proxy_placeholder(name: str):
-            return self.create_proxy('placeholder', name, (), {},
+            param = sig.parameters[name]
+            default = () if param.default is inspect.Parameter.empty else (param.default,)
```

Comment:
Another way to do this is to pull `__defaults__` from the object, but I think this should be okay

Reply:
If I just use `default = () if not fn_for_analysis.__defaults__ else fn_for_analysis.__defaults__`, then the code would fail in the case of multiple default args. I could get around this with some sort of global counter, but I don't see a more elegant solution. What do you think?